pkg/test/framework/components/istio/operator.go

```diff
@@ -227,6 +247,17 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster kube.Cluster,
 		// This MUST match the clusterName in the remote secret for this cluster.
 		installSettings = append(installSettings, "--set", "values.global.multiCluster.clusterName="+cluster.Name())
 	}
+	if cluster.SharesControlPlane() {
+		// enable mesh expansion so remote clusters can use the control plane
+		installSettings = append(installSettings, "--set", "values.global.meshExpansion.enabled=true")
+	}
+	if cluster.IsControlPlaneCluster() {
+		installSettings = append(installSettings, "--set", "profile=default")
```

Comment:
Is this needed? I would just leave it unset so that any tests can override it if needed